src/main/java/Duke.java

```diff
@@ -1,10 +1,260 @@
+import myexceptions.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+
+
+    static ArrayList<Task> tasks = new ArrayList<Task>();
+    static int taskPosition = 0;
+    static String description;
+    static String by;
+    static int indexOfBy;
+
+
+    private static void writeToFile(String filePath, ArrayList<Task> tasks) throws IOException{
+        FileWriter fw = new FileWriter(filePath);
+
+        for(Task t:tasks) {
+            fw.write( t.getStatusIcon() + " " + t.getDescription() + "\n");
+
+        }
+        fw.close();
+    }
+
+    private static void restoreFileContents(String filePath) throws FileNotFoundException, BlankDescriptionException {
+        File f = new File(filePath);
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            inputToTasks(s.nextLine());
+        }
+    }
+
+    private static void inputToTasks(String nextLine) throws BlankDescriptionException {
+        int taskStatusIndex = 1;
+        int taskDescriptionIndex = 7;
+
+        char taskType = nextLine.charAt(1);
+        String taskStatus = nextLine.substring(4,5);
+        String taskDescription = nextLine.substring(7);
+        try {
+            switch (taskType) {
+            case 'T':
```

Comment:
Naming a task type as a single character is [ambiguous](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#not-too-long-not-too-short).  How about trying other names that could directly describe the task types?